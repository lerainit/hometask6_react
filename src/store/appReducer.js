import { combineReducers } from "redux";
import cardIndexReducer from "./cardId/reducer";
import modalReducer from './modal/reducer'
import modalTextReducer from "./modalText/reducer";
import CartReducer from "./addCards/reducer";
import addFavoritesReducer from "./addFavorites/reducer";
import productsReducer from "./products/reducer";


const appReducer = combineReducers({
   modal:modalReducer,
   cardIndex:cardIndexReducer,
   modalText:modalTextReducer,
   Cart:CartReducer,
   addFavorites:addFavoritesReducer,
   products:productsReducer

})
export default appReducer