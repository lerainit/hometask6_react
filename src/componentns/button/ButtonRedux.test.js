import Button from "./button";
import{render} from '@testing-library/react'
import { Provider } from "react-redux";
import store from '../../store/'



describe('Button renders', () => {

    test('should Button match snapshot', () => {
        const{asFragment} = render(<Button/>)
        expect(asFragment()).toMatchSnapshot()

    })

})
