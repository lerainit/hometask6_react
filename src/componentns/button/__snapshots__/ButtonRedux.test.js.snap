// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders should Button match snapshot 1`] = `
<DocumentFragment>
  <button
    class=""
  >
    Button
  </button>
</DocumentFragment>
`;
